lib/cfn_nag.rb

```diff
@@ -206,9 +206,11 @@ def generic_json_rules(input_json, rule_directories)
     rule_directories.each do |rule_directory|
       rules = Dir[File.join(rule_directory, '*.rb')].sort
 
-      rules.each do |rule_file|
-        @input_json = input_json
-        eval IO.read(rule_file)
+      if rules.length > 0
```

Comment:
If a rule_directory is specified, but that directory doesn't contain any .rb files that can be eval'd, it blows up with an error.